bin/src/file_creator_helper.dart

```diff
@@ -50,20 +50,22 @@ class FileCreatorHelper {
   static void createScreenFile(String projectName, String screenName,
       {required bool generateDI}) {
     final sb = StringBuffer()
-      ..writeln(
-          "import 'package:$projectName/viewmodel/$screenName/${screenName}_viewmodel.dart';")
-      ..writeln(
-          "import 'package:$projectName/widget/provider/provider_widget.dart';")
       ..writeln("import 'package:flutter/material.dart';");
     if (generateDI) {
       sb.writeln("import 'package:get_it/get_it.dart';");
     }
     sb
+      ..writeln("import 'package:$projectName/navigator/route_names.dart';")
```

Comment:
first add all imports to a list. After that sort the list. And then you can do the `..writeln()`